tests/console/validate_multipath.pm

```diff
@@ -0,0 +1,172 @@
+# SUSE's openQA tests
+#
+# Copyright © 2020 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved. This file is offered as-is,
+# without any warranty.
+
+# Summary: Validate multipathing on the installed system.
+# The multipathing support in SUSE Linux Enterprise Server is based on
+# the Device Mapper Multipath module of the Linux kernel and the multipath-tools user space package.
+# the Multiple Devices Administration utility (multipath) can be used to view the status of multipathed devices.
+#
+# Maintainer: QA SLE YaST team <qa-sle-yast@suse.de>
+
+use strict;
+use warnings;
+use base "opensusebasetest";
+use testapi;
+use scheduler 'get_test_suite_data';
+use Test::Assert ':all';
+
+sub verify_packages_installed {
+    my ($pkg_ref) = shift;
+    record_info('pkgs', 'Verify required packages installed');
+    assert_script_run("rpm -qa | grep $_",
+        fail_message => "$_ is not installed in the system") for (@{$pkg_ref});
+}
+
+sub verify_kernel_modules_loaded {
+    my ($mod_ref) = shift;
```

Comment:
it is enough clear by the signature of the function.

Reply:
@jknphy i think this is a mindful request. it is always better to avoid this shortcuts in the names when ever is not necessary. i dont think i need to analyze why for it and it is something that you request also in some of your review. Could you reconsider the changes on the names please?

Reply:
this is a thing that depends on the case. The fact that I have requested changes in your PRs is not a reason for you to insist here. I might be wrong on that time and it might have been a different context.
I like it as it is, because you read the method and you know that you are verifying modules loaded, it is a good practice to not save words on the methods, but in the variables when is super clear it doesn't make any sense, if the code would be refactored or would longer than it would apply to have a better name or full name. I think I'm being consist here in my code and in my explanation about it.

Reply:
i dont insist because the reason that you mention. I have done this mistake myself. The reason is readability and maintainability. Someone else wants to check and read our code after sometime. he might search for keywords to find about it. it should be as much self-explanatory as it can by its own.

Reply:
how could help for someone to search `mod_ref` or `modules_ref` ... ? I don't get your point.